src/libs/OptionsListUtils.js

```diff
@@ -502,27 +504,47 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
 
         const lastMessageTextFromReport = getLastMessageTextForReport(report);
         const lastActorDetails = personalDetailMap[report.lastActorAccountID] || null;
-        let lastMessageText = hasMultipleParticipants && lastActorDetails && lastActorDetails.accountID !== currentUserAccountID ? `${lastActorDetails.displayName}: ` : '';
-        lastMessageText += report ? lastMessageTextFromReport : '';
-
+        const displayName = hasMultipleParticipants && lastActorDetails && lastActorDetails.accountID !== currentUserAccountID ? `${lastActorDetails.displayName}: ` : '';
+
+        let alternateText = _.isString(lastMessageTextFromReport)
+            ? `${displayName}${lastMessageTextFromReport}`
+            : {
+                  ...lastMessageTextFromReport,
+                  transformer: (text) => `${displayName}${lastMessageTextFromReport.transformer ? lastMessageTextFromReport.transformer(text) : text}`,
```

Comment:
This part of the code is not much readable, if we can simplify it it would be great.

Reply:
bump @tienifr, any chance we can simplify this? I'm worried we're sacrificing readability here to fix a minor bug.